### PR TITLE
FLUID-5678: Updated docs to reflect changes in HTML and stylus for UI…

### PR DESCRIPTION
**Pull request by maozillah**
Updated docs to reflect changes in [Pull request 611](https://github.com/fluid-project/infusion/pull/611)

### CODE REVIEW

File: src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingPanels.md [hunk 1]
```diff
@@ -110,17 +110,35 @@ fluid.defaults("fluid.prefs.panel.lineSpace", {
...
         multiplier: ".flc-prefsEditor-multiplier"
     },
+    selectors: {
+            lineSpace: ".flc-prefsEditor-line-space",
+            label: ".flc-prefsEditor-line-space-label",
+            narrowIcon: ".flc-prefsEditor-line-space-narrowIcon",
+            wideIcon: ".flc-prefsEditor-line-space-wideIcon",
+            multiplier: ".flc-prefsEditor-multiplier",
+            lineSpaceDescr: ".flc-prefsEditor-linespace-descr"
+    },
```
Comment (jobara) [inline src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingPanels.md]: There are two selectors blocks now. I think you meant to replace the old one. Also the tabbing is off in the new one.

**Comment by jobara** [comment]
@maozillah for the API changes, could you make sure to note any selectors that have changed (e.g. choiceLabel to tocDesc). The same goes for any other option that has changed.

**Comment by maozillah** [comment]
Updated pull request

File: src/documents/APIChangesFrom1_5To2_0.md [hunk 2]
```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
...
+* `.flc-prefsEditor-contrast-descr`
+* `.flc-prefsEditor-linespace-descr`
+* `.flc-prefsEditor-textsize-descr`
+* `.flc-prefsEditor-textfont-descr`
+
+######Changes######
+<table>
+<tr><th> 1.5</th><th>2.0</th></tr>
+<tr>
+    <td><code>flc-prefsEditor-toc-choice-label </code></td>
```
Comment (jobara) [inline src/documents/APIChangesFrom1_5To2_0.md]: extra white space after label

Comment (maozillah) [inline src/documents/APIChangesFrom1_5To2_0.md]: Removed

File: src/documents/APIChangesFrom1_5To2_0.md [hunk 3]
```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
...
+</tr>
+</table>
+
+##### CSS Selectors #####
+
+######Additions######
+* `.flc-prefsEditor-contrast-descr`
+* `.flc-prefsEditor-linespace-descr`
+* `.flc-prefsEditor-textsize-descr`
+* `.flc-prefsEditor-textfont-descr`
```
Comment (jobara) [inline src/documents/APIChangesFrom1_5To2_0.md]: You should also have the selector name here. The messageBundle keys and selectors don't have to be the same, although they are in this case. Also, it's not clear what these map to. I'd do something like

* `textSizeDescr: ".flc-prefsEditor-textsize-descr"`

or 

```javascript
textSizeDescr: ".flc-prefsEditor-textsize-descr"
```

The same sort of thing should be in the changes section where you should show the selector name and the css selector used.

File: src/documents/APIChangesFrom1_5To2_0.md [hunk 4]
```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
...
+    <td><code>tocChoiceLabel</code></td>
+    <td><code>tocDescr</code></td>
+</tr>
+<tr>
+    <td><code>simplifyChoiceLabel</code></td>
+    <td><code>simplifyDescr</code></td>
+</tr>
+</table>
+
+##### CSS Selectors #####
```
Comment (jobara) [inline src/documents/APIChangesFrom1_5To2_0.md]: @acheetham what do you think about this section being called "CSS Selectors". I'm a bit worried that it may be confused with CSS styling rather than Javascript DOM selection.

File: src/documents/tutorial-creatingANewAdjusterUI/CreatingANewAdjusterUI.md [hunk 5]
```diff
@@ -220,20 +226,21 @@ For the radio buttons:
 
 ![a checkbox styled table of contents panel](../images/checkbox-styled.png)
 
+**Note:** Currently the description label must be placed before the on/off toggle to prevent spacing issues from occuring. This will be addressed with a restyling of the on/off toggle with more details at the [FLUID-5708 JIRA](https://issues.fluidproject.org/browse/FLUID-5708)
```
Comment (jobara) [inline src/documents/tutorial-creatingANewAdjusterUI/CreatingANewAdjusterUI.md]: Can you make the entire note emphasized. e.g. _**Note:** some note_

File: src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html [hunk 6]
```diff
@@ -3,9 +3,9 @@
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
+    	<span class="flc-prefsEditor-speak-descr"></span>
         <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
             <span class="fl-prefsEditor-switch-inner"></span>
         </span>
-        <span class="flc-prefsEditor-speak-choice-label"></span>
```
Comment (jobara) [inline src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html]: I don't think you meant to change this file, it's actually part of the the Infusion package that is being pulled in to the documentation site. If something needs to be changed the entire infusion library should be updated.

File: src/documents/APIChangesFrom1_5To2_0.md (hunk 4 above)
Comment (acheetham) [inline src/documents/APIChangesFrom1_5To2_0.md]: @jobara, I was initially going to agree, and say we should just call the section "Selectors" since it's refering to an option called "selectors", but then I had a look at the file as a whole to be sure I knew what I was talking about, and I realized I can't tell what this section is saying. The "CSS Selectors" heading is a sibling to the "Message Bundle Keys and Selectors" – what is the difference in selectors? The "CSS Selectors" heading is followed by "additions" and a list of selectors, but what are they? Don't our DOM selectors have names? What are the names for these selectors? Am I even right that this is referring to the 'selectors' option?

File: src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html (hunk 6 above)
Comment (maozillah) [inline src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html]: Gotcha, I think this change is in the [FLUID-5678](https://github.com/fluid-project/infusion/pull/611) pull request

**Comment by maozillah** [comment]
Updated, hopefully more clear now.

File: src/documents/APIChangesFrom1_5To2_0.md [hunk 7]
```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
...
+    <td><code>choiceLabel: {messagekey: "speakChoiceLabel"}</code></td>
+    <td><code>speakDescr: {messagekey: "speakDescr"}</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "tocChoiceLabel"}</code></td>
+    <td><code>tocDescr: {messagekey: "tocDescr"}</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: {messagekey: "simplifyChoiceLabel"}</code></td>
+    <td><code>simplifyDescr: {messagekey: "simplifyDescr"}</code></td>
```
Comment (jobara) [inline src/documents/APIChangesFrom1_5To2_0.md]: I don't think it is necessary to show the `{messagekey: "..."}` in the changes and additions for the message bundles section. The `messagekey` is specific to the prototree. The messages could be accessed through the prototree, msgLookup and etc. I think simply listing the keys here would be fine (same in the Additions section. You could however consider listing any changes to the text in the default bundles, but I'm not really sure that is necessary either.

File: src/documents/APIChangesFrom1_5To2_0.md [hunk 8]
```diff
@@ -10,6 +10,59 @@ This page contains a list of the features, APIs, and etc. that have changed in I
...
+    <td><code>choiceLabel: ".flc-prefsEditor-toc-choice-label"</code></td>
+    <td><code>tocDescr: ".flc-prefsEditor-toc-descr"</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: ".flc-prefsEditor-speak-choice-label"</code></td>
+    <td><code>speakDescr: ".flc-prefsEditor-speak-descr"</code></td>
+</tr>
+<tr>
+    <td><code>choiceLabel: ".demo-prefsEditor-simplify-choice-label"</code></td>
+    <td><code>simplifyDescr: ".demo-prefsEditor-simplify-descr"</code></td>
```
Comment (jobara) [inline src/documents/APIChangesFrom1_5To2_0.md]: The "demo" ones don't need to be included in this document. They aren't actually part of any supported API.

File: src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html (hunk 6 above)
Comment (jobara) [inline src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html]: You should make sure to revert the change to this file, in this pull request.

**Comment by maozillah** [comment]
Updated

File: src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html [hunk 9]
```diff
@@ -6,6 +6,6 @@
...
+        <span class="flc-prefsEditor-speak-descr"></span>
```
Comment (jobara) [inline src/static/lib/infusion/src/framework/preferences/html/PrefsEditorTemplate-speak.html]: looks like you still have a change in this file.

**Comment by maozillah** [comment]
Updated

File: src/documents/APIChangesFrom1_5To2_0.md [hunk 10]
```diff
@@ -10,6 +10,55 @@ This page contains a list of the features, APIs, and etc. that have changed in I
 
 ### Preferences Framework ###
 
+#### Panel Changes ####
+Changes and additions are being made to UI Options in order to improve its compliance with accesibility standards.
```
Comment (jobara) [inline src/documents/APIChangesFrom1_5To2_0.md]: Maybe you could rephrase this as "Changes and additions are being made to compliance with accesibility standards." because it is a change for the Preferences Framework in general.

**Comment by maozillah** [comment]
Update

**Comment by jobara** [comment]
@maozillah while you're making changes in this file, could you please fix the link in the note under the [Component Grade Changes](https://github.com/maozillah/infusion-docs/blob/FLUID-5678/src/documents/APIChangesFrom1_5To2_0.md#component-grade-changes) section. Because it is in a markup block, it needs to use standard HTML for the link rather than the markdown syntax.

**Comment by maozillah** [comment]
Updated

**Comment by jobara** [comment]
Closed in favour of https://github.com/fluid-project/infusion-docs/pull/72 which updates the contents of this pull request to address merge conflicts.
